pagopa/io-web-profile
# [IOPID-628] :  404 static error page

## Short description

Created 404 error page because the default next-js page (not-found.js) does not generate the static page with output: export. Here is the link to the open issue on next: https://github.com/vercel/next.js/issues/54239. So we created the page in src/app/[locale]/(pages)/404/page.tsx

## List of changes proposed in this pull request

- New static 404 error page
- Changed href for the footer links

## How to test

- yarn run build
check that in the  folder out/it  exist the static 404.html page

-yarn run start-static
 go to /404
